### PR TITLE
DEVDOCS-4425: [edit] Partner sandbox guide, add global regions

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4425](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4425)

## What changed?
Added global regions to partner sandbox creation

### CODE REVIEW

**Review by bc-tgomez — DISMISSED** [review]
Looks good to me!

**Comment by bc-traciporter** [comment]
@slsriehl I love the images; however, they have been changing the navigation menus fairly often so this might cause us more work in the long run.

File: docs/api-docs/partner/create-a-sandbox.md [hunk 1]
```diff
@@ -1,45 +1,113 @@
...
-1. Navigate to the [BigCommerce Partner Portal](https://partners.bigcommerce.com/English/) and sign in using your credentials.
-2. After signing in, click **Create New**, and select one of the 2 options:
-    * **Deal Registration w/store**: Create a sandbox with a deal registration.
-    * **Partner Use Sandbox**: Create a sandbox for your own internal use.
-3. If you select **Deal Registration w/ Store**, fill in all the fields on the **Create a Store/Register a Deal** page.
-    * In the **Create a Store** field, select **Partner Sandbox**.
-    * Click **Submit**.
- 4. If you select **Partner Use Sandbox**, fill in the **Sandbox Name** field, and then click **Submit**.
+1. Navigate to the [BigCommerce Partner Portal](https://partners.bigcommerce.com/) and sign in using your credentials.
+2. After signing in, hover over **Create New** in the top nav, then select one of the following two options:
```
Comment (bc-traciporter) [inline docs/api-docs/partner/create-a-sandbox.md:12]:
```suggestion
2. After signing in, hover over **Create New** in the top navigation, then select one of the following two options:
```

File: docs/api-docs/partner/create-a-sandbox.md [hunk 2]
```diff
@@ -1,45 +1,113 @@
...
+<!-- theme: warning -->
+> #### Store regions are immutable
+> You cannot update the store region of a partner sandbox.
 
-If you select the **Deal Registration w/Store** option, you will receive two emails: a Deal Registration Confirmation email, and your new sandbox Store Login Credentials email, containing the credentials to your sandbox. If you select the **Partner Sandbox** option, you will receive an email with the sandbox credentials only. 
-Note that the email containing the sandbox credentials is sent only once. You can view all your related stores by signing into your account with these credentials.
+### Deal Registration sandbox
 
-You can view your newly created sandbox by navigating to **Reporting > Review Sandboxes & Trials** on the Partner Portal.
+After you click **Create New > Deal Registration** in the top nav, complete the following steps:
```
Comment (bc-traciporter) [inline docs/api-docs/partner/create-a-sandbox.md:24]:
```suggestion
After you click **Create New > Deal Registration** in the top navigation, complete the following steps:
```

File: docs/api-docs/partner/create-a-sandbox.md [hunk 3]
```diff
@@ -1,45 +1,113 @@
...
+
+5. Click **Submit**.
+
+After you click **Submit**, you will receive two emails: 
+* a Deal Registration confirmation email, and 
+* an email that contains sign in credentials for the new sandbox.
+
+For more about viewing and working with partner sandboxes, consult the following section on [accessing and managing partner sandboxes](#accessing-and-managing-partner-sandboxes).
+  
+### Partner Use sandbox
```
Comment (bc-traciporter) [inline docs/api-docs/partner/create-a-sandbox.md:48]:
```suggestion
### Partner use sandbox
```

File: docs/api-docs/partner/create-a-sandbox.md [hunk 4]
```diff
@@ -1,45 +1,113 @@
...
+
+For more about viewing and working with partner sandboxes, consult the following section on [accessing and managing partner sandboxes](#accessing-and-managing-partner-sandboxes).
+
+## Accessing and managing partner sandboxes
+
+Allow up to 15 minutes for the new store to appear in any list of stores.
+
+The email that contains the sandbox credentials is sent only once. However, you can access the sandbox by signing in to the [BigCommerce account](https://login.bigcommerce.com/accounts/list) that uses the same email address as your Partner Portal account.
+
+To review and manage your sandbox and trial stores in the Partner Portal, hover over **Reporting** in the top nav, then click **Review Sandboxes & Trials**.
```
Comment (bc-traciporter) [inline docs/api-docs/partner/create-a-sandbox.md:74]:
```suggestion
To review and manage your sandbox and trial stores in the Partner Portal, hover over **Reporting** in the top navigation, then click **Review Sandboxes & Trials**.
```

File: docs/api-docs/partner/create-a-sandbox.md [hunk 5]
```diff
@@ -1,45 +1,113 @@
...
-3. Select the type of issue you have (Product Support or Billing Support).
-4. Select **Yes** to the question "Is this related to a specific store?"
-5. Enter your new sandbox store’s canonical URL (for example: http://your-store-name.mybigcommerce.com). 
-6. Sign into your new store using the same email associated with your partner login.
-7. Click **Help** in the Admin panel and scroll down to select your desired **Services & Support**. 
+To create a partner-specific support case, complete the following steps:
 
-## Resources
+1. Sign in to the [Partner Portal](https://partners.bigcommerce.com). 
+2. In the top nav, hover over **Support**, then click **Contact Support**.
```
Comment (bc-traciporter) [inline docs/api-docs/partner/create-a-sandbox.md:85]:
```suggestion
2. In the top navigation, hover over **Support**, then click **Contact Support**.
```

**Review by bc-traciporter — COMMENTED** [review]
I changed nav to navigation but feel free to keep it as is.

File: docs/api-docs/partner/create-a-sandbox.md (hunk 3 above)
Comment (slsriehl) [reply docs/api-docs/partner/create-a-sandbox.md:48]: i'm gonna decline this one

**Comment by slsriehl** [comment]
@bc-traciporter all nav changed to navigation.

**Review by slsriehl — APPROVED** [review]
teamwork, hooray!

**Pull request merged**